proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/HealthCheckAppRuleTests.java

```diff
@@ -1,29 +1,43 @@
 package uk.gov.ida.notification.apprule;
 
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.Appender;
 import io.dropwizard.testing.ConfigOverride;
 import io.dropwizard.testing.junit.DropwizardClientRule;
 import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.slf4j.LoggerFactory;
 import uk.gov.ida.notification.apprule.base.GatewayAppRuleTestBase;
 import uk.gov.ida.notification.apprule.rules.GatewayAppRule;
 import uk.gov.ida.notification.apprule.rules.TestEidasSamlResource;
 import uk.gov.ida.notification.apprule.rules.TestTranslatorResource;
 import uk.gov.ida.notification.apprule.rules.TestVerifyServiceProviderResource;
+import uk.gov.ida.notification.shared.logging.ProxyNodeLogger;
 
+import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
 
+@RunWith(MockitoJUnitRunner.class)
 public class HealthCheckAppRuleTests extends GatewayAppRuleTestBase {
 
     private static final String METADATA_PUBLISH_PATH = "/proxy-node-md-publish-path";
-    private static final String METADATA_RESOURCE_PATH = "metadata/test-proxy-node-metadata.xml";
+    private static final String METADATA_File_PATH =
+            HealthCheckAppRuleTests.class.getClassLoader().getResource("metadata/test-proxy-node-metadata.xml").getPath();
```

Comment:
Unusual capitalisation. Not really important to change, though!

Reply:
Ugh I'll fix it in the next PR